CHANGELOG.md

```diff
@@ -9,35 +9,36 @@
   - to_timestamp_ltz
   - to_timestamp_ntz
   - to_timestamp_tz
+  - locate
 - Added support for ASOF JOIN type.
 - Added support for the following local testing APIs:
-  - to_timestamp
-  - to_timestamp_ltz
-  - to_timestamp_ntz
-  - to_timestamp_tz
-  - greatest
-  - least
-  - dateadd
-- Added support for ASOF JOIN type.
-- Added support for `locate` in `snowflake.snowpark.functions`.
-- Added support for the following local testing APIs:
-  - Session.get_current_account
-  - Session.get_current_warehouse
-  - Session.get_current_role
-  - Session.use_schema
-  - Session.use_warehouse
-  - Session.use_database
-  - Session.use_role
+  - snowflake.snowpark.functions:
+    - to_timestamp
+    - to_timestamp_ltz
+    - to_timestamp_ntz
+    - to_timestamp_tz
+    - greatest
+    - least
+    - dateadd
+  - snowflake.snowpark.Session:
+    - get_current_account
+    - get_current_warehouse
+    - get_current_role
+    - use_schema
+    - use_warehouse
+    - use_database
+    - use_role
 
 ### Bug Fixes
 
 - Fixed a bug in `SnowflakePlanBuilder` that `save_as_table` does not filter column that name start with '$' and follow by number correctly.
-- Fixed a bug in local testing implementation of LEFT ANTI and LEFT SEMI joins where rows with null values are dropped.
-- Fixed a bug in local testing implementation of DataFrameReader.csv when the optional parameter `field_optionally_enclosed_by` is specified.
-- Fixed a bug in local testing implementation of Column.regexp where only the first entry is considered when `pattern` is a `Column`.
-- Fixed a bug in local testing implementation of Table.update in which null value in the rows to be updated causes `KeyError`.
-- Fixed a bug in local testing implementation where VARIANT columns raise errors at `DataFrame.collect`.
-- Fixed a bug in local testing implementation of `count_distinct`.
+- Fixed bugs in local testing:
+  - LEFT ANTI and LEFT SEMI joins where rows with null values are dropped.
```

Comment:
I think the grammar needs to be changed too then, each of the nested bullet points should be a description of the error behavior, e.g. `Column.regexp only considers the first entry when pattern is a Column.`

Reply:
I attempted to update them, I will let @sfc-gh-mabrennan do a second round polishing